Wrappers/Python/cil/optimisation/functions/Function.py

```diff
@@ -25,13 +25,16 @@
 
 class Function(object):
     
-    """ Abstract class representing a function 
+    r""" Abstract class representing a function 
     
-        :param L: Lipschitz constant of the gradient of the function F(x), when it is differentiable.
-        :type L: number, positive, default None
-        :param domain: The domain of the function.
+        Parameters
+        ----------
+        
+        L: number, positive, default None
+            Lipschitz constant of the gradient of the function F(x), when it is differentiable.
+        domain: The domain of the function.
 
-        Lipschitz of the gradient of the function; it is a positive real number, such that |f'(x) - f'(y)| <= L ||x-y||, assuming f: IG --> R
+        Lipschitz of the gradient of the function; it is a positive real number, such that :math:`\|f'(x) - f'(y)\| <= L \|x-y\|`, assuming :math:`f: IG \rightarrow \mathbb{R}`
```

Comment:
Is the :param supposed to be there in the bottom line?
![image](https://github.com/TomographicImaging/CIL/assets/77114597/524a8bf0-b387-48f4-af2c-72bf60f3c108)

Reply:
Thanks Hannah

Reply:
Looks good to me